packages/components/src/experience-control/index.tsx

```diff
@@ -1,97 +1,126 @@
-import { Button } from '@wordpress/components';
+import { BaseControl, useBaseControlProps } from '@wordpress/components';
 import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
 import { IconBad, IconGood, IconNeutral } from './icons';
 
 import './style.scss';
 
-enum Experience {
+export enum ExperienceType {
 	GOOD = 'good',
 	NEUTRAL = 'neutral',
 	BAD = 'bad',
 }
 
+type ExperienceOption = {
+	value: ExperienceType;
+	icon: JSX.Element;
+	ariaLabel: string;
+};
+
+interface ExperienceControlOptionProps {
+	className?: string;
+	checked: boolean;
+	onClick: () => void;
+	children: React.ReactNode;
+	value: string;
+	name: string;
+	ariaLabel: string;
+}
+
 const ExperienceControlOption = ( {
 	className,
-	isSelected,
+	checked,
 	onClick,
 	children,
-}: {
-	className?: string;
-	isSelected: boolean;
-	onClick: () => void;
-	children: React.ReactNode;
-} ) => (
-	<Button
-		className={ clsx( 'experience-control__button', className, {
-			'is-selected': isSelected,
+	value,
+	name,
+	ariaLabel,
+}: ExperienceControlOptionProps ) => (
+	<label
+		className={ clsx( 'experience-control__option', className, {
+			'is-selected': checked,
 		} ) }
-		onClick={ onClick }
 	>
-		<div className="experience-control__button-content">{ children }</div>
-	</Button>
+		<input
+			type="radio"
+			className="experience-control__radio"
+			checked={ checked }
+			onChange={ onClick }
+			value={ value }
+			name={ name }
+			aria-label={ ariaLabel }
+		/>
+		<div className="experience-control__option-content">{ children }</div>
+	</label>
 );
 
-const ExperienceControlBase = ( {
-	label,
-	children,
-	helpText,
-}: {
-	label: string;
+interface ExperienceControlBaseProps {
 	children: React.ReactNode;
-	helpText?: string;
-} ) => (
-	<div className="experience-control">
-		<div className="experience-control__label">{ label }</div>
-		<div className="experience-control__buttons">{ children }</div>
-		{ helpText && <div className="experience-control__help-text">{ helpText }</div> }
-	</div>
-);
+	label?: string;
+	help?: string;
+}
+
+const ExperienceControlBase = ( { children, ...props }: ExperienceControlBaseProps ) => {
+	const { baseControlProps } = useBaseControlProps( props );
+
+	return (
+		<BaseControl className="experience-control" { ...baseControlProps }>
+			<div className="experience-control__options" role="radiogroup">
+				{ children }
+			</div>
+		</BaseControl>
+	);
+};
 
-export function ExperienceControl( {
-	label,
-	onChange,
-	selectedExperience,
-	helpText,
-}: {
+interface ExperienceControlProps {
 	label: string;
-	onChange: ( experience: string ) => void;
-	selectedExperience: string;
-	helpText?: string;
-} ) {
-	const handleChange = ( experience: string ) => {
-		onChange( experience );
-	};
+	onChange: ( experience: ExperienceType ) => void;
+	value: ExperienceType;
+	help?: string;
+}
```

Comment:
Some of these can be `Pick<>`ed off of the types we already defined, which will keep things DRY (especially when they include TSDoc descriptions).